src/plots/barPlot.ts

```diff
@@ -584,7 +585,9 @@ namespace Plottable.Plots {
         };
         writer.write(text, containerWidth, containerHeight, writeOptions);
 
-        let tooWide = this._isVertical ? barWidth < measurement.width : barHeight < measurement.height;
+        let tooWide = this._isVertical
+          ? barWidth < (measurement.width + Bar._LABEL_PADDING * 2)
```

Comment:
maybe a comment to help clarify that we want the padding applied on _both_ sides of the text? otherwise lgtm